integreat_cms/cms/templates/languages/language_form.html

```diff
@@ -79,5 +79,24 @@ <h3 class="font-bold text-white"><i data-feather="settings" class="mr-2"></i> {%
             </div>
         </div>
     </div>
+    </div>
+    {% if form.instance.id and perms.cms.delete_language %}
```

Comment:
Should we add the permission `delete_language` in the migrations for at least one role?

Reply:
I'm not sure, as this function is probably very rarely used, I wouldn't have a problem with an administrator having to do this.
But since languages are protected from deletion when used in a region, I don't see a problem with giving the delete permission to the municipality team either.
Maybe we should keep this in mind and do a general permission refactoring when everything is settled? I'd avoid having too many back-and-forth migrations for the permissions...